src/viur/shop/modules/order.py

```diff
@@ -40,11 +42,34 @@ def current_session_order_key(self, value: db.Key) -> None:
         self.session["session_order_key"] = value
         current.session.get().markChanged()
 
+    @property
+    def current_order_skel(self) -> SkeletonInstance_T[OrderSkel] | None:
+        if not self.current_session_order_key:
+            return None
+        return self.order_get(self.current_session_order_key)
+
     # --- (internal) API methods ----------------------------------------------
 
     @exposed
-    def payment_providers_list(self):
-        return JsonResponse(get_payment_providers_list())
+    def payment_providers_list(  # TODO(discuss): Move: to API?
+        self,
+        only_available: bool = True,
```

Comment:
@ArneGudermann Or should we name this `ignore_available` or `list_all`, ?

Reply:
I think `only_available` is a good name.

Reply:
okay, can you review this PR and #36?